sdcm/cluster_gce.py

```diff
@@ -347,6 +347,8 @@ def set_tags_as_labels(_instance):
 
         self.log.info('Created %s instance %s', 'spot' if spot else 'on-demand', instance)
         try:
+            general_tag = {"created_by": "sct"}
```

Comment:
the `test_id` will also be very useful

Reply:
We already have it.
This is to tag something as default for the cases the bottom one fails.